modules/installation-ibm-power-vs-config-yaml.adoc

```diff
@@ -109,51 +114,56 @@ compute: <1> <2>
 - architecture: ppc64le
   hyperthreading: Enabled <3>
   name: worker
-  platform: {}
+  platform:
+    powervs:
+      smtLevel: 4 <4>
   replicas: 3
 controlPlane: <1> <2>
   architecture: ppc64le
   hyperthreading: Enabled <3>
   name: master
-  platform: {}
+  platform:
+    powervs:
+      smtLevel: 4 <4>
   replicas: 3
 metadata:
   creationTimestamp: null
   name: example-private-cluster-name
 networking:
   clusterNetwork:
-  - cidr: 10.128.0.0/14 <4>
+  - cidr: 10.128.0.0/14 <5>
     hostPrefix: 23
   machineNetwork:
   - cidr: 192.168.0.0/24
-  networkType: OVNKubernetes <5>
+  networkType: OVNKubernetes <6>
   serviceNetwork:
   - 172.30.0.0/16
 platform:
   powervs:
     userID: ibm-user-id
     powervsResourceGroup: "ibmcloud-resource-group"
     region: powervs-region
-    vpcName: name-of-existing-vpc <6>
+    vpcName: name-of-existing-vpc <7>
     cloudConnectionName: powervs-region-example-cloud-con-priv
     vpcSubnets:
     - powervs-region-example-subnet-1
     vpcRegion : vpc-region
     zone: powervs-zone
     serviceInstanceID: "powervs-region-service-instance-id"
-publish: Internal <7>
-pullSecret: '{"auths": ...}' <8>
-sshKey: ssh-ed25519 AAAA... <9>
+publish: Internal <8>
+pullSecret: '{"auths": ...}' <9>
+sshKey: ssh-ed25519 AAAA... <10>
 ----
 <1> If you do not provide these parameters and values, the installation program provides the default value.
 <2> The `controlPlane` section is a single mapping, but the compute section is a sequence of mappings. To meet the requirements of the different data structures, the first line of the `compute` section must begin with a hyphen, `-`, and the first line of the `controlPlane` section must not. Both sections currently define a single machine pool. Only one control plane pool is used.
 <3> Whether to enable or disable simultaneous multithreading, or `hyperthreading`. By default, simultaneous multithreading is enabled to increase the performance of your machines' cores. You can disable it by setting the parameter value to `Disabled`. If you disable simultaneous multithreading in some cluster machines, you must disable it in all cluster machines.
-<4> The machine CIDR must contain the subnets for the compute machines and control plane machines.
-<5> The cluster network plugin to install. The supported values are `OVNKubernetes` and `OpenShiftSDN`. The default value is `OVNKubernetes`.
-<6> Specify the name of an existing VPC.
-<7> How to publish the user-facing endpoints of your cluster. Set publish to `Internal` to deploy a private cluster.
-<8> Required. The installation program prompts you for this value.
-<9> Provide the `sshKey` value that you use to access the machines in your cluster.
+<4> The SMTLevel specifies the level of SMT to set to the control plane and compute machines. The supported values are from 1 to 8.
```

Comment:
You cannot set it to any value between 1 and 8.  For e.g. 3, 5, 6, 7 etc.  Only Powers of 2 are allowed, 1, 2, 4, 8.